src/coreComponents/constitutive/solid/SolidBase.cpp

```diff
@@ -39,30 +39,44 @@ SolidBase::SolidBase( string const & name, Group * const parent ):
   registerWrapper( viewKeyStruct::stressString(), &m_newStress ).
     setPlotLevel( PlotLevel::LEVEL_0 ).
     setApplyDefaultValue( 0 ). // default to zero initial stress
-    setDescription( "Current Material Stress" ).
+    setDescription( "Current Material Stress [Pa]" ).
```

Comment:
This is actually not true. GEOS allows for any consistent set of units. There is no need to force the use to use SI. Everything should be agnostic to the unit system used.

Reply:
done